### ISSUE TITLE
Plain text readme (for game)

### ISSUE THREAD

**Issue by RattleSN4K3** [UltraStar-Deluxe/USDX#78]
As of 5cadefc the markdown README.md file is stating only "Development" and not the version anymore. Additionally some things are changed to support the markdown styling. This makes the plain text _unreadable_ (see tables for instance).  By fixing #75, there likey will be a readme but it has to be updated.

**Comment by RattleSN4K3** [comment]
With the update of the NSIS installer (397970bef00d74222ef8638e2629906b9d38eebb), the readme was restored. Currently, the installer will include the Readme.txt. The Readme.md seems to be too specific for this repo and the source code and not the game version of the project. Not sure if this stays like that. An additional readme would need curation and therefore someone needs to update it.

**Comment by DJaeger** [comment]
As @basisbit updated the installer to use the README.md in f0c0da6 is the README.txt still needed?
As the README.md is still using tables and such things the topic has not really changed, right?

**Comment by RattleSN4K3** [comment]
As a player/user, I'd prefer a plain text readable version of any readme file. The README.md does already contain non-game specific stuff (but also in my opinion too much non-developement related bits of information) and as a developer/tester various project specific things are missing (like how to contribute; CONTRIBUTE.md).

I had a different opinion on the case of readme files. The latest changes weren't my idea. I don't know how to continue with this issue as I am not aware what basisbit (or any other related dev) has in mind. In my opinion, an additionally readme.txt (as installer-docs) wouldn't be too much in keeping it up-to-date. Granted, the README.md and installer-readme-txt-file would share alot of information but I would at least convert the README.md to a proper readable format (like README.htm, or even txt) and keeping it up-to-date at least for every release.

**Comment by DJaeger** [comment]
I would suggest to restore the previous readme.txt, update it und move it to the installer directory.

**Comment by basisbit** [comment]
what for? nowadays, everyone can read up-to-date websites / wiki pages. That .txt file would just add additional work to keep it current and be outdated already a few days after a release. There are reasons why current games don't come with manuals any more.
But whatever, if you think that a .txt file is the best solution, go ahead and add it again. But don't be surprised if the file is not updated by developers for years.

**Comment by DJaeger** [comment]
Then it should be removed completely and shouldn't be replaced by the readme.md

**Comment by basisbit** [comment]
you mean remove it from the install folder? That might be a good idea indeed

**Comment by dgruss** [comment]
i don't think this would be a good idea, as there is some info on game configuration in the README.md as well as a link to the wiki.
